### ISSUE TITLE
Version warning does not appear as top banner

### ISSUE THREAD

**Issue by hugovk** [readthedocs/sphinx_rtd_theme#1027]
### Problem

Sometimes people read/link to old versions of our documentation and miss new features and changes.

For example, from a Google search for a module, I ended up at http://pillow.readthedocs.org/en/3.1.x/reference/ImageChops.html

RtD has a setting to help with this:

> ## Version warning
>
> This is a banner that appears on the top of every page of your docs that aren’t stable or latest. This banner has a text with a link redirecting the users to the latest version of your docs.
>
> This feature is disabled by default on new projects, you can enable it in the admin section of your docs (`Admin` > `Advanced Settings`).

[docs.readthedocs.io/en/stable/versions.html#version-warning](https://docs.readthedocs.io/en/stable/versions.html#version-warning)

However this is enabled at [readthedocs.org/dashboard/pillow/advanced](https://readthedocs.org/dashboard/pillow/advanced/) and whilst we *do* get a red bar on the version selector at the bottom left, there's no top banner with a redirect to the latest version:

![image](https://user-images.githubusercontent.com/1324225/102853555-3b0d3000-4429-11eb-860b-7b04cd958d1a.png)

### Reproducible Project

https://pillow.readthedocs.io/en/3.1.x/

#### Expected Results

I expect to see "a banner that appears on the top of every page of your docs that aren’t stable or latest. This banner has a text with a link redirecting the users to the latest version of your docs."

### Environment Info

- Python Version: 3.7
- Sphinx Version: 3.4.0
- RTD Theme Version: 0.4.3

**Comment by stsewd** [comment]
Hi, looks like that version isn't active https://readthedocs.org/projects/pillow/versions/, I guess it didn't get deleted from storage from some reason.

**Comment by hugovk** [comment]
Yep, we only have `latest` and `stable` active, all the rest are inactive.

At least these are inactive but still online:

* https://pillow.readthedocs.io/en/3.0.x/
* https://pillow.readthedocs.io/en/3.1.x/
* https://pillow.readthedocs.io/en/4.0.x/
* https://pillow.readthedocs.io/en/4.1.x/
* https://pillow.readthedocs.io/en/4.2.x/
* https://pillow.readthedocs.io/en/5.1.x/
* https://pillow.readthedocs.io/en/7.0.x/
* https://pillow.readthedocs.io/en/7.1.x/
